hw/ip/otbn/dv/otbnsim/sim/state.py

```diff
@@ -403,7 +403,6 @@ def stop(self) -> None:
         clear the busy flag and write STOP_PC.
 
         '''
-
```

Comment:
Nit: Since you're not touching this file otherwise, I'd probably drop this hunk.